src/home/Footer.jsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import "./Footer.css";
+import { Link } from "react-router-dom";
+import github from "./../assets/images/github.svg";
+import linkdin from "./../assets/images/LinkedIn.svg";
+import twitter from "./../assets/images/Twitter.svg";
+import facebook from "./../assets/images/facebook.svg";
+export default function Footer() {
+  return (
+    <section id="footer">
+      <div className="footer-container">
+        <div className="main-flexbox">
+          <div className="flexbox-item">
+            <img className="logo" src="#" alt="Logo" />
+            <h4 className="text">
+              We are known for connecting top talents with top companies. We
+              make great possibilities happen.
+            </h4>
+          </div>
+          <div className="flexbox-item">
+            <h3 className="heading">Explore</h3>
+            <ul className="list">
+              <Link to="/" style={{ textDecoration: "none" }}>
+                <li className="li">Home</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">About Us</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">Blogs</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">FAQ</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">Contact Us</li>
+              </Link>
+            </ul>
+          </div>
+          <div className="flexbox-item">
+            <h3 className="heading">Visit</h3>
+            <ul className="list">
+              <li className="li">3 Birrel Avenue</li>
+              <li className="li">Ashish</li>
+              <li className="li">Bangladesh, India</li>
+            </ul>
+          </div>
+          <div className="flexbox-item">
+            <h3 className="heading">Legal</h3>
+            <ul className="list">
+              <li className="li">
+                <a href="https://anitab.org/terms-of-use/" className="Links" target="_blank" rel="noopener noreferrer"  >
+                  Terms
+                </a>
+              </li>
+              <li className="li">
+                <a href="https://anitab.org/privacy-policy/" className="Links" target="_blank" rel="noopener noreferrer">
+                  Privacy
```

Comment:
```suggestion
                  Privacy Policy
```

Reply:
> > I just have one minor request. Can you please set it so that homepage content is maintained (so not partially cut) by the footer?
> > This is also because later on when we added all the sctions of the Homepage as per the Figma file, the Footer needs to be fixed at the very bottom.
> 
> @naveen8801 Please fix this as well

@Rahulm2310 @mtreacy002 ,  I think its working fine now , i have added footer component after Section element in Home.jsx due to which home page properly covering 80vh of height .